packages/aws-client/S3.js

```diff
@@ -621,3 +622,70 @@ exports.getFileBucketAndKey = (pathParams) => {
  */
 exports.createBucket = (Bucket) =>
   awsServices.s3().createBucket({ Bucket }).promise();
+
+/**
+ * Copy an S3 object to another location in S3 using a multipart copy
+ *
+ * @param {Object} params
+ * @param {string} params.sourceBucket
+ * @param {string} params.sourceKey
+ * @param {string} params.destinationBucket
+ * @param {string} params.destinationKey
+ * @returns {Promise<undefined>}
+ */
+const multipartCopyObject = async (params) => {
```

Comment:
nit: Do we want to consider naming this `uploadCopy` to show its correspondence to both the `upload` and `uploadPartCopy` functions that are already part of the SDK? Just a thought on name consistency. I'm not bothered either way.

Reply:
I thought about that but decided that it would be more confusing. It's already pretty confusing that AWS makes you refer to the "S3 to S3" copy operations as "upload" operations. I'd rather insulate the users of this function from that confusion if possible.

Reply:
Works for me.